IdentityCore/src/MSIDError.m

```diff
@@ -29,6 +29,7 @@
 NSString *MSIDCorrelationIdKey = @"MSIDCorrelationIdKey";
 NSString *MSIDHTTPHeadersKey = @"MSIDHTTPHeadersKey";
 NSString *MSIDHTTPResponseCodeKey = @"MSIDHTTPResponseCodeKey";
+NSString *MSIDAdditionalUserIdentifierKey = @"MSIDAdditionalUserIdentifierKey";
```

Comment:
nit: can you call it MSIDUserDisplayableIdkey?
We added this key in latest common core, I'd like those two to be the same in future.